src/Axiom/Set.agda

```diff
@@ -167,13 +167,13 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   ∈-unions : {a : A} {U : Set (Set A)} → (∃[ T ] T ∈ U × a ∈ T) ⇔ a ∈ proj₁ (unions U)
   ∈-unions = proj₂ $ unions _
 
-  ∅ : Set A
-  ∅ = fromList []
+  ﹛﹜ : Set A
```

Comment:
These are small curly braces obtained from `\(` (or `\)`) and then hitting `C-f` a bunch of times.